projects/angular/src/forms/styles/_password.clarity.scss

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2016-2023 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+@use '../../utils/mixins';
+@use '../../utils/variables/variables.global';
+@use 'mixins.forms' as forms-mixins;
+@use 'variables.forms' as forms-variables;
+
+.clr-input-wrapper {
+  input[clrPassword] {
```

Comment:
I would write this like the following so that `input[clrPassword]` isn't repeated several time.

```scss
.clr-input-wrapper input[clrPassword] {
  ...

  & + button.clr-input-group-icon-action {
    ...
  }

  & + button.clr-input-group-icon-action[disabled] {
    ...
  }

  &[disabled] {
    ...
  }
}
```